homeassistant/components/ezviz/camera.py

```diff
@@ -109,11 +110,23 @@ async def async_setup_entry(hass, entry, async_add_entities):
             )
 
         else:
+
+            await hass.config_entries.flow.async_init(
+                DOMAIN,
+                context={"source": SOURCE_DISCOVERY},
+                data={
+                    ATTR_SERIAL: camera[ATTR_SERIAL],
+                    CONF_USERNAME: None,
+                    CONF_PASSWORD: None,
+                },
+            )
+
             camera_username = DEFAULT_CAMERA_USERNAME
             camera_password = ""
             camera_rtsp_stream = ""
-            _LOGGER.info(
-                "Found camera with serial %s without configuration. Please add an integration instance to see the camera stream",
+            ffmpeg_arguments = DEFAULT_FFMPEG_ARGUMENTS
+            _LOGGER.warning(
+                "Found camera with serial %s without configuration. Please go to integration to complete setup.",
```

Comment:
We don't end logging messages with period.

Reply:
Done.